ethereum/statetransition/build.gradle

```diff
@@ -27,6 +27,7 @@ dependencies {
 
   implementation 'io.tmio:tuweni-units'
   implementation 'org.hyperledger.besu.internal:metrics-core'
+  implementation 'io.libp2p:jvm-libp2p'
```

Comment:
nit: I believe at some point we would need to extract a big portion of classes from the statetransition module to somewhere else

Reply:
I was thinking how to transition these changes to `teku:master` and we couldn't do it in one batch, it's probably to do it feature by feature, so it could be reviewed. Classes could be moved during this stage. It will require a lot of refactoring anyway.